2_adventures/001/xcmer/05/hangman.rb

```diff
@@ -0,0 +1,84 @@
+require 'set'
+
+DICT_PATH = '/usr/share/dict/words'
+
+def get_words_count(dict)
+  count = 0
+  File.open(dict) {|f| count = f.read.count("\n") }
+end
+
+def get_random_word(dict, total_words)
+  number = Random.rand(total_words)
+  current = 0
+
+  File.foreach(dict) {|x|
```

Comment:
each_line or readlines may be what you want to avoid the newline. (I noticed you used chomp in line 16.)

The methods to look at is in IO class.